astropy/wcs/__init__.py

```diff
@@ -1,21 +1,24 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 """
 .. _wcslib: http://www.atnf.csiro.au/~mcalabre/WCS/
-.. _Paper IV: http://www.atnf.csiro.au/people/mcalabre/WCS/index.html
+.. _WCS papers: http://www.atnf.csiro.au/people/mcalabre/WCS/index.html
+.. _Paper IV: http://www.atnf.csiro.au/people/mcalabre/WCS/dcs_20040422.pdf
```

Comment:
Yes, the numbering is confusing, especially to someone who comes to this for the first time and doesn't know the history. Related to this I just noticed that the "time " paper (the current Paper IV) is not on Calabretta's site. I wonder if it's not better when referring to "FITS standard" to link to this page:

http://fits.gsfc.nasa.gov/fits_wcs.html

which looks a bit more complete and is probably maintained more regularly. It also has a link to Calabretta's site.